code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -279,24 +280,29 @@
 
 		//cache for sanic speed
 		var/datum/excited_group/enemy_excited_group = enemy_tile.excited_group
-		//If we are both in an excited group, and they aren't the same, merge.
+		//If we are both in an excited group, and they aren't the same, merge your group into ours
```

Comment:
This comment isn't accurate, merging isn't really directional, it goes based off the action that takes the least amount of work